a/ansible.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: This is a hello-world example
+  hosts: ansibleclient01.local
+  tasks:
+    - name: Create a file called '/tmp/testfile.txt' with the content 'hello world'.
```

Comment:
Where is the "Hello World" content added?

Reply:
Would you be okay with me just creating a new PR for just LÖVE? Unless you want me to write out all of the steps for this one, but it's a lot. 🙂

Reply:
You do what works best for you.